content/applications/sales/sales/products_prices/products/variants.rst

```diff
@@ -1,225 +1,178 @@
 
-- Quality: T-Shirts, Polos, Shirts
+To create a new attribute, click :guilabel:`Create`. First, choose an attribute name, such as
+"Color" or "Size."
 
-- Color: Red, Blue
+.. image:: variants/attribute-creation.png
+   :align: center
+   :alt: Attribute creation window.
 
-- Size: S, M, L, XL
+Then, select a :guilabel:`Display Type`, which determines how this product will be shown on the
+eCommerce page, Point of Sale dashboard, and Product Configurator.
 
-In such a use case, you could create 1 template with three dimensions of
-variants (Layout, T-Shirts, Polos). But, it's recommended to create two
-different product templates as T-shirts may highly differ from polos or
-shirts and customer expect to see these as two different products in the
-e-Commerce:
+- :guilabel:`Radio`: Options appear in a bullet style list.
+- :guilabel:`Select`: Options appear in a dropdown menu.
+- :guilabel:`Color`: Options appear as small colored squares, which reflect any HTML color codes
+  set.
 
-- Product Template: T-shirt
+.. image:: variants/display-types-configurator.png
+   :align: center
+   :alt: Display Types on Product Configurator.
 
-  - Color: Red, Blue
-  - Size: S, M, L, XL
+The :guilabel:`Variants Creation Mode` informs Odoo when to automatically create a new variant once
+an attribute is added to a product.
 
-- Product Template: Polos
+- :guilabel:`Instantly`: Creates all possible variants as soon as attributes and values are added
+  to a product template.
+- :guilabel:`Dynamically`: Creates variants only when corresponding attributes and values are added
+  to a sales order.
+- :guilabel:`Never`: Never automatically creates variants.
 
-  - Color: Red, Blue
-  - Size: S, M, L, XL
+.. note::
+   Once added to a product, an attribute's :guilabel:`Variants Creation Mode` cannot be edited.
 
-Configuration
-=============
+Values should be added to an attribute before saving, but more values can be added at any time,
+if needed. To add a value, click :guilabel:`Add a line`. From there, you can:
 
-Activate the variant feature
-----------------------------
+#. Type in the value's name.
+#. Check whether or not the value is custom (i.e. the customer provides unique specifications).
+#. Specifically for colors, add an HTML color to make it even easier for salespeople and customers
+   to know what they're selecting.
 
-Before you can use product variants, you must first activate the product
-variants in the settings. To do so, you must go to the Sales app. In the
-menu :menuselection:`Configuration --> Settings`, locate the **Products Variants** line,
-and tick the option **Products can have several attributes**, then click
-on **Apply**.
+A color code can be selected either by dragging the slider around or by entering a specific HTML
+color code (e.g. #FF0077).
 
-.. image:: variants/variant01.png
+.. image:: variants/picking-a-color.png
    :align: center
+   :alt: Selecting a color.
 
```

Comment:
Add a paragraph: `` Finally, click :guilabel:`Save` to finish creating the attribute. ``